blog/modules/ROOT/pages/sort-antora-components.adoc

```diff
@@ -0,0 +1,196 @@
+= Awesome Antora: Sort and hide components and versions
+David Jencks <djencks@apache.org>
+:page-revdate: 2020-9-28 14:25
+:page-author: David Jencks
+:description: You would like custom sorting of Antora components and versions, or to hide components.
+:page-tags: Antora, Sorting Components
+:page-image: sort.jpg
+:page-color-primary: #8BB3B2
+:page-color-secondary: #C68B45
+:ms-sorter-url: https://gitlab.com/opendevise/oss/antora-site-generator-ms/-/blob/v2.3.x/lib/custom-sort-components.js
+
+A frequent request is to sort the components and their versions in the default Antora UI component explorer in a customized order.
+Fortunately, this is quite easy to do using a couple of files in a supplemental UI or in a customized UI.
+I'd like to thank Bob Roro and Ewan Edwards for pushing me into figuring out how this works and a lot of hints and answers and Dan Allen for pointing to an link:{ms-sorter-url}[existing implementation] (for a completely custom UI).
```

Comment:
This sentence is a bit long no? 
At least for a non-native speaker, I get lost at "... how this works and a lot of hints and answers and..." with the double "and" 😅 
I don't understand to what "a lot of hints and answers" is attached?

Reply:
How about...
```
I'd like to thank:

* Bob Roro and Ewan Edwards for pushing me into figuring out how this works and a lot of hints about what to do and answers about how to do it
* Dan Allen for pointing to an link:{ms-sorter-url}[existing implementation] (for a completely custom UI).
```

Reply:
I feel like a verb is missing, if we duplicate the sentence:

- Bob Roro and Ewan Edwards for pushing me into figuring out how this works (OK)
- Bob Roro and Ewan Edwards for pushing me into figuring out a lot of hints about what to do and answers about how to do it (???)

What about:

I'd like to thank Bob Roro and Ewan Edwards for giving me a lot of hints about what to do and answers about how to do it. With their invaluable help, they pushed me into figuring out how this works!
I'd also like to thank Dan Allen for pointing to an existing implementation (for a completely custom UI).

Reply:
I like the 3 sentences.  I think the middle sentence might be more idiomatic in french.... I suggest...

I'd like to thank Bob Roro and Ewan Edwards for a lot of hints about what to do and answers about how to do it.
Their invaluable help pushed me into figuring out how this works!
I'd also like to thank Dan Allen for pointing to an existing implementation (for a completely custom UI).

Reply:
All good, your suggestion is great :+1: